### PR TITLE
chore: merge master from getredash/redash (ENG-3127)

### PR DESCRIPTION
**Pull request by thisisshi**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] New Query Runner (Data Source)
- [ ] New Alert Destination
- [ ] Other

### CODE REVIEW

**Review by tvansteenburgh — COMMENTED** [review]
Kind of impossible to do a meaningful review on this since it's so huge. Can you comment on how it was tested?

**Comment by thisisshi** [comment]
@tvansteenburgh I've been deploying to my sandbox and monitoring the redash dashboard in cloudwatch, running queries and looking for/fixing any tracebacks i see in the server, worker, scheduled worker, and installer tasks

**Comment by thisisshi** [comment]
also @tvansteenburgh here's a diff of this branch vs getredash/redash's master branch: https://github.com/getredash/redash/compare/master...stacklet:redash:sonny/stacklet/integration

this should be a better diff to look at

File: redash/models/users.py [hunk 1]
```diff
@@ -85,30 +84,25 @@ class User(
...
-    _profile_image_url = Column("profile_image_url", db.String(320), nullable=True)
     password_hash = Column(db.String(128), nullable=True)
     group_ids = Column(
-        "groups", MutableList.as_mutable(postgresql.ARRAY(key_type("Group"))), nullable=True
+        "groups",
+        MutableList.as_mutable(ARRAY(key_type("Group"))),
+        nullable=True,
     )
     api_key = Column(db.String(40), default=lambda: generate_token(40), unique=True)
-    db_role = Column(db.String(128), nullable=True)
```
Comment (johnsca) [inline redash/models/users.py:94]: This PR seems to be deleting some of our customizations.

**Pull request closed without merging**